README.md

```diff
@@ -29,12 +29,12 @@ Microsoft PowerToys is a set of utilities for power users to tune and streamline
 - Windows 11 or Windows 10 v1903 (18362) or newer.
```

Comment:
```suggestion
 - Supported Operating Systems:
   - Windows 10 v1903 (18362) or newer.
   - Windows 11 (all versions)
```

This looks better and eliminates the multiple `or` words.

Reply:
Not a part of the release notes, so I wouldn't touch it for this PR.